R/SSexecutivesummary.r

```diff
@@ -1099,7 +1116,7 @@ SSexecutivesummary <- function(replist,
         comma(tot.bio.all, 0),
         comma(ssb.all, 0),
         comma(smry.all, 0),
-        print(depl.all * 100, 1),
+        print(depl.all, 3),
```

Comment:
I can but I think I will wait for @chantelwetzel-noaa's vote on this because I was trying to minimize changes that didn't directly have to do with a 🐛 to make the pull request easier to review.

Reply:
I agree about consulting @chantelwetzel-noaa on stuff like this. 
I'm happy to merge this PR sooner and put this piece and the directory issue into separate issues for dealing with later, or leave this open until she's available.

Reply:
I am fully on board moving away from depletion (depl.all) in the code.  I try to never use depletion in the document text so it would be an improvement to have that same logic carried through in the code.